src/components/custom/SdkCompatibilityTable/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import type { SdkCompatibility } from '@site/src/datasets/types';
+import { features } from '@site/src/datasets/constants';
+import { Popover } from '@headlessui/react';
+import { faCircleExclamation } from '@fortawesome/free-solid-svg-icons';
+
+import Link from '@docusaurus/Link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export default function SdkCompatibilityTable({ compatibility }: { compatibility: SdkCompatibility[] }) {
```

Comment:
I'm sure this component could be optimized but the dataset is so small I didn't invest the time. If anyone has a suggestion, please let me know and I would be happy to test it out.